.github/workflows/ci.yml

```diff
@@ -44,8 +44,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-        with:
-          submodules: 'true'
```

Comment:
On the npm package repo (https://www.npmjs.com/package/@open-inwoner/design-tokens/v/0.0.3-alpha.0) you say this (my emphasis):

If you are using Github actions, **add** these to your script:
```
      - uses: actions/checkout@v3
        with:
          submodules: 'true'
```
Here you're removing the same part. Is this correct?

Reply:
Now that that entire project is an NPM package (which will just render the `dist/css`, so _not_ the style-dictionary module), I do not expect anyone will still use the external package as a submodule. This is of course still possible - but _adding it as a submodule is only practical for development, if people want to change the tokens_; so the submodule is not for implementation.

I can add that to the Readme there later.